client/app/js/crypto/module.js

```diff
@@ -63,7 +67,7 @@ angular.module('GLBrowserCrypto', [])
   }
 
   return {
-    restrict: 'E',
+    restrict: 'A',
```

Comment:
generally is preferrable to use standard html elements (e.g. <div>) and try to design our elements so that are compatible with html (e.g. lets see for our custom if we can go for the data-* that pass the validators/wai-aria stuff)